agent/pkg/dagent/README.md

```diff
@@ -2,24 +2,6 @@
 
 Implement the .NET Mandy in Go
```

Comment:
Invalid too, lol.

Reply:
@nandor-magyar suggested/approved these changes when were debugging.

Reply:
Sorry, it was a bit confusing, but I refer to deleting the 3rd line too.